exotica_python/src/Exotica.cpp

```diff
@@ -596,7 +596,7 @@ PYBIND11_MODULE(_pyexotica, module)
     kdlFrame.def("getTranslationAndAngleAxis", [](KDL::Frame* me) { return getFrameAsVector(*me, RotationType::ANGLE_AXIS); });
     kdlFrame.def("getTranslationAndQuaternion", [](KDL::Frame* me) { return getFrameAsVector(*me, RotationType::QUATERNION); });
     kdlFrame.def("getFrame", [](KDL::Frame* me) { return getFrame(*me); });
-    kdlFrame.def("inverse", (KDL::Frame(KDL::Frame::*)() const) & KDL::Frame::Inverse);
+    kdlFrame.def("inverse", (KDL::Frame (KDL::Frame::*)() const) & KDL::Frame::Inverse);
```

Comment:
This diff will need to go away by a rebase and merge of conflicts (already fixed in master)